src/components/Tabs/Tabs.jsx

```diff
@@ -103,12 +118,21 @@ const Tabs = React.createClass(createLucidComponentDefinition({
 							<li
 								className={boundClassNames('Tab', {
 									'Tab-is-active': index === actualSelectedIndex,
-									'Tab-is-active-and-open': isOpen && index === actualSelectedIndex
+									'Tab-is-disabled': tabChildProp.isDisabled,
+									'Tab-is-active-and-open': isOpen && index === actualSelectedIndex,
+									'Tab-is-progressive': isProgressive && index !== tabChildProps.length - 1,
 								})}
 								key={index}
-								onClick={_.partial(onSelect, index)}
+								onClick={_.partial(onSelect, index, tabChildProp)}
 							>
 								{_.get(_.first(Tabs.Title.findInAllAsProps(tabChildProp)), 'children', '')}
+								{isProgressive && index !== tabChildProps.length - 1 ?
+										<svg className={boundClassNames('Tab-arrow')} xmlns='http://www.w3.org/2000/svg' width='8px' height='28px' viewBox='0 0 8 28' >
```

Comment:
Had it that way originally but because of an issue with the docs we can not have more than one component in a file.  Since it would only be used in the design it does not make sense to have a separate file.